spanner/api/Spanner.Samples/CreateDatabaseAsync.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START spanner_create_database]
+
+using Google.Cloud.Spanner.Data;
+using System.Threading.Tasks;
+
+public class CreateDatabaseAsyncSample
+{
+    public async Task CreateDatabaseAsyncAsync(string projectId, string instanceId, string databaseId)
+    {
+        string connectionString = $"Data Source=projects/{projectId}/instances/{instanceId}";
+
+        using (var connection = new SpannerConnection(connectionString))
+        {
+            string createStatement = $"CREATE DATABASE `{databaseId}`";
+            var cmd = connection.CreateDdlCommand(createStatement);
+            await cmd.ExecuteNonQueryAsync();
+        }
+        // Update connection string with Database ID for table creation.
+        connectionString = connectionString + $"/databases/{databaseId}";
+        using (var connection = new SpannerConnection(connectionString))
+        {
+            // Define create table statement for table #1.
+            string createTableStatement =
+           @"CREATE TABLE Singers (
+                     SingerId INT64 NOT NULL,
+                     FirstName    STRING(1024),
+                     LastName STRING(1024),
+                     ComposerInfo   BYTES(MAX)
```

Comment:
nit: The spacing here is all different.
```suggestion
                     SingerId INT64 NOT NULL,
                     FirstName STRING(1024),
                     LastName STRING(1024),
                     ComposerInfo BYTES(MAX)
```